CONTRIBUTING.md

```diff
@@ -1 +1,39 @@
 # Contributing to Real-time Ubuntu
+
+The [Real-time Ubuntu documentation] lives in the `docs/` directory and is deployed on Read the Docs.
+It is written using the [reStructuredText] format and built using [Sphinx].
+
+The documentation should follow the [style guide] in use for documentation at Canonical.
+
```

Comment:
## Download and install the docs

If you are working with this documentation set for the first time, you'll need to create a fork of the [real-time-ubuntu-docs] repository on your GitHub account, and clone the forked repository.

Reply:
This is obvious. Adding more text makes the reader (stop reading and) ignore other important information, such as the style guide and make recipes.

Reply:
Hi @farshidtz , actually I had this exact same question when I first wanted to contribute to another repository.
I think we need to consider new and experienced contributors.
So we signpost the docs (with proper headers / sections) so if it's irrelevant the user just skips to the `make install`  section. But for new users they would understand the workflow (to fork & clone) without having to look elsewhere.

Reply:
Hi @farshidtz , please review the newly-added suggestions only. The rest of the previous comments are no longer relevant.

Thanks.